ui/design-system-core/src/components/ModerationSwitchCard/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { tw } from '@twind/core';
+
+import Text from '../Text';
+
+export interface IModerationSwitchCardProps {
+  tabs: { title: string; value: string }[];
+  activeTab: string;
+  onTabClick: Dispatch<SetStateAction<string>>;
+}
+
+const ModerationSwitchCard: React.FC<IModerationSwitchCardProps> = props => {
+  const { tabs, activeTab, onTabClick } = props;
+
+  const handleTabClick = (value: string) => {
+    onTabClick(value);
+  };
+
+  return (
+    <div className={tw('flex w-full')}>
+      {tabs.map((tab, idx) => (
+        <div
+          key={tab.title + idx}
+          className={tw(
+            `w-[25%] p-4 border-b ${
+              tab.value === activeTab
+                ? ' border-secondary-light dark:border-secondary-dark'
+                : 'border-grey8 dark:border-grey3'
```

Comment:
you can use the [grouping syntax](https://twind.style/grouping-syntax) like border(grey8 dark:grey3)